manticore/ethereum/manticore.py

```diff
@@ -464,9 +399,13 @@ def __init__(self, workspace_url: str = None, policy: str = "random"):
         super().__init__(initial_state, workspace_url=workspace_url, policy=policy)
         self.subscribe("will_terminate_state", self._terminate_state_callback)
         self.subscribe("did_evm_execute_instruction", self._did_evm_execute_instruction_callback)
-        self.subscribe("did_read_code", self._did_evm_read_code)
-        self.subscribe("on_symbolic_sha3", self._on_symbolic_sha3_callback)
-        self.subscribe("on_concrete_sha3", self._on_concrete_sha3_callback)
+        consts.sha3 = consts.sha3.symbolicate
```

Comment:
Is this meant to be hardcoded here? With it in place, it looks like the next `if` statement will always be false and the one following it will always be true.